<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <widget class="QPushButton" name="touchButton_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>No! Not Again!</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="TimeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0 Step</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="touchButton_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>I told you
DON'T TOUCH</string>
    </property>
   </widget>
   <widget class="QPushButton" name="touchButton_3">
    <property name="geometry">
     <rect>
      <x>-100</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Haha! My Hide
is PERFECT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>...Fine. I don't hide anymore. But I assure you, this is no more than an ordinary edit box... Don't be so upset bro :)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionPause_2"/>
    <addaction name="actionContinue_2"/>
    <addaction name="separator"/>
    <addaction name="actionRestart_2"/>
    <addaction name="separator"/>
    <addaction name="actionSave_2"/>
    <addaction name="actionLoad_2"/>
   </widget>
   <widget class="QMenu" name="menuHey_What_s_this">
    <property name="title">
     <string>NOTHING here</string>
    </property>
    <addaction name="actionDon_t_touch"/>
    <addaction name="actionnew_npy"/>
    <addaction name="actionbecome_rich"/>
    <addaction name="actionDon_t_touch_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHey_What_s_this"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionPause_2">
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionContinue_2">
   <property name="text">
    <string>Continue</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionRestart_2">
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave_2">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_2">
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionDon_t_touch">
   <property name="text">
    <string>set GPA to 4.0</string>
   </property>
  </action>
  <action name="actionnew_npy">
   <property name="text">
    <string>new npy = Npy()</string>
   </property>
  </action>
  <action name="actionbecome_rich">
   <property name="text">
    <string>become rich</string>
   </property>
  </action>
  <action name="actionDon_t_touch_2">
   <property name="text">
    <string>Don't touch</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
